users/app.py

```diff
@@ -54,22 +54,54 @@ def add_update_user():
     return jsonify(user_object), (201 if added_new_user else 200)
 
 
+def upsert_user_in_db(user_object, users_collection):
+    """Updates or inserts the user object into users_collection.
+
+    Sets/overwrites default values as needed.
+
+    user_object should look like
+        {"user_id": foobar,
+         "name": Foo Bar}
+
+    Documents in users_collection should look like
```

Comment:
I'm not sure you want to document the schema here; if you do so, you'll need to update this comment if you change the schema in the future.

Reply:
Done https://github.com/knative-portability/large-events/pull/8/commits/89632bcea2b0d94f886c3a9f5b4d03cc940150e2